messages/scheduler/scheduler.fr-BE.yml

```diff
@@ -4,13 +4,13 @@ kendo:
     dailyInterval: jour(s)
 
     # The text similar to 'Repeat every' displayed in the recurrence editor.
-    dailyRepeatEvery: 'Répéter chaque:'
+    dailyRepeatEvery: 'Répéter chaque :'
```

Comment:
Is it common to have a space before the colon?

Reply:
In French, yes.